### DISCUSSION TITLE
Binding to model properties not working as expected

### DISCUSSION THREAD

**Discussion by zoomyboy** [livewire/livewire#8948] [1. Help]
In the Livewire V2 presentation a few years ago, Caleb demonstrated how to wire:model-bind directly to Eloquent model properties.
That way I dont need to reference the properties all over the place (in the Mmount method, in validation, yada yada)..

This is also documented in the V2 docs:
https://laravel-livewire.com/docs/2.x/properties#binding-models

Why exactly has this been disabled in V3 and what would be the "V3 way" of doing it? I'm feeling that this is a step back which makes code again more verbose. 

https://livewire.laravel.com/docs/upgrading#eloquent-model-binding

**Comment by EdgarsJoja** [comment]
Your own provided link answers one of your questions it seems. The livewire 3 "way" would be:
> In Livewire 3, binding directly to Eloquent models has been disabled in favor of using individual properties, or extracting [Form Objects](https://livewire.laravel.com/docs/forms#extracting-a-form-object).

and there seems to be also backwards compatibility toggle:
> By setting legacy_model_binding to true, Livewire will handle Eloquent model properties exactly as it did in version 2.

**Reply by francoism90** [reply]
Also note legacy model binding is not recommended, and basically unsupported (if I understood this correctly).

Form Objects are great and a better alternative for wire:model binding.
